lib/index.ts

```diff
@@ -29,6 +33,34 @@ export function useStoryblokApi(): StoryblokClient {
   return globalThis.storyblokApiInstance;
 }
 
+export async function useStoryblok({
+  slug,
```

Comment:
Hi @dipankarmaikap regarding this, I would usually prefer passing an object with named parameters due to the benefits you mention on Slack (like empty parameters) but I would prefer if we keep the consistency of the API with the other SDKs https://github.com/storyblok/storyblok-vue/blob/c1627b33db24de408a4a39c18e4f1a9398e071fa/lib/index.ts#L53-L57